### PR TITLE
Add more examples

### PR DESCRIPTION
**Pull request by Jerry-py**
## Description

Add more examples
## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [x] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: examples/message_commands.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from EpikCord import Client, Intents, StringOption, NumberOption
```
Comment (TheUntraceable) [inline examples/message_commands.py:1]: Remove the 2 unused imports, `StringOption` and `NumberOption`

File: examples/slash_commands.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import asyncio
+import datetime
+import time
+from EpikCord import Client, Intents, StringOption, NumberOption
```
Comment (TheUntraceable) [inline examples/slash_commands.py:4]: Remove the 2 unused imports, `StringOption` and `NumberOption`

File: examples/user_commands.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from EpikCord import Client, Intents, StringOption, NumberOption
```
Comment (TheUntraceable) [inline examples/user_commands.py:1]: Remove the 2 unused imports, `StringOption` and `NumberOption`

**Review by TheUntraceable — CHANGES_REQUESTED** [review]
Just remove all the unused imports and I'll merge

**Comment by Jerrydotpy** [comment]
Busy rn can you do it for me? When your finished editing the files - don't merge PR because there a PR in my fork about optimizing images. I'll do that at home (2:00 or later UTC-4)

**Comment by Jerry-py** [comment]
You can merge now @TheUntraceable

**Pull request merged**